modules/uid2IdSystem_shared.js

```diff
@@ -151,6 +352,94 @@ export class Uid2StorageManager {
   }
 }
 
+class UID2CstgBox {
```

Comment:
What is a CstgBox? This looks like it's a CstgCryptoProvider or something like that - I don't think `Box` tells me anything about what it is.

Reply:
https://github.com/IABTechLab/uid2-web-integrations/blob/main/src/uid2CstgBox.ts
this is refer to the file here, do we want to make them consistent?

Reply:
I named it after NaCl's [`crypto_box`](https://nacl.cr.yp.to/box.html).